tests/unit/accounts/test_views.py

```diff
@@ -170,7 +170,7 @@ def test_post_validate_redirects(self, monkeypatch, pyramid_request,
 
         assert isinstance(result, HTTPSeeOther)
 
-        assert result.headers["Location"] == "/"
+        assert result.headers["Location"] == "/manage/projects"
```

Comment:
This kind of introduces the same problem as my previous comment: if this path ever changes, we'll have to come and update this string to patch it.

Since we're not actually interested in testing that `route_path` works as expected, what we usually do in these tests is mock it out like so:

```py
pyramid_request.route_path = pretend.call_recorder(lambda a: '/the-redirect')
```

And then later, we can assert that it was called with the right route identifier:

```py
assert pyramid_request.route_path.calls == [pretend.call('manage.projects')]
```

And that the response has the path we stubbed out:

```py
assert result.headers["Location"] == "/the-redirect"
```